src/PrestaShopBundle/Form/Admin/Improve/Shipping/Carrier/ShippingLocationsAndCostsType.php

```diff
@@ -57,45 +78,58 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ])
             ->add('is_free', SwitchType::class, [
                 'label' => $this->trans('Free Shipping', 'Admin.Shipping.Feature'),
-                'external_link' => [
```

Comment:
Why remove this, although the link was missing I believe there is a section in the shop settings about the free shipping no?

Reply:
I don't find it 🤔 Do you know where?